Source/WebCore/Modules/filesystemaccess/FileSystemHandle.h

```diff
@@ -61,6 +61,7 @@ class FileSystemHandle : public ActiveDOMObject, public RefCounted<FileSystemHan
 protected:
     FileSystemHandle(ScriptExecutionContext&, Kind, String&& name, FileSystemHandleIdentifier, Ref<FileSystemStorageConnection>&&);
     FileSystemStorageConnection& connection() { return m_connection.get(); }
+    Ref<FileSystemStorageConnection> protectedConnection();
```

Comment:
Why do we need `protectedConnection()`? `connection()` is inlined and returns a `const` data member after your PR, we shouldn't need  to both mark the data member as const *and* a protected function.

Reply:
Oops, I think I only realized it could be `const` later but then I forgot to revisit my earlier changes. This also applied to `filesystem()`.